tests/ext/segfault_backtrace_enabled.phpt

```diff
@@ -4,8 +4,11 @@ Dump backtrace when segmentation fault signal is raised and config enables it
 <?php
 if (!extension_loaded('posix')) die('skip: posix extension required');
 if (getenv('SKIP_ASAN') || getenv('USE_ZEND_ALLOC') === '0') die("skip: intentionally causes segfaults");
+if (getenv('DD_TRACE_CI_PECL') === '1') die("skip: pecl run-tests does not support XFAIL");
 if (file_exists("/etc/os-release") && preg_match("/alpine/i", file_get_contents("/etc/os-release"))) die("skip Unsupported LIBC");
 ?>
+--XFAIL--
```

Comment:
This handles the case where the same fault causes phpunit or run-tests.php to fail. We do not see this in CI, it's nevertheless a possibility.